src/governance.rst

```diff
@@ -0,0 +1,172 @@
+Governance
+==========
+This document outlines the policies and procedures that manage the conda-forge
+community. This document recognizes that, while packaging is deeply important
+to all of us, conda-forge is an all-volunteer organization. Members of the
+core, any team, or external contributors choose to participate and may
+choose to leave at any time for any reason or for no reason. We deeply
+appreciate all good faith contributions.
+
+Code of Conduct
+---------------
+Conda-forge adheres to the
+`Numfocus Code of Conduct <https://www.numfocus.org/code-of-conduct>`_.
+
+Teams & Roles
+-------------
+Here are defined the primary teams participating in conda-forge activities.
+
+* **core:** The core team is the governing body over the entire conda-forge
+  organization. Members of the core team have full rights over all conda-forge
+  repositories. Members of core are the face of the project, and are responsible
+  for officially interfacing with external communities, organizations, non-profits,
+  and companies. The core team may create new sub-teams, as appropriate.
+  Each member of core is entitled to one vote on all elected matters.
+* **staged-recipes:** The staged-recipes team administers the staged-recipes
+  repository and is responsible for the review and creation of new feedstocks.
```

Comment:
Should we include the helper teams?

cc @CJ-Wright

Reply:
Maybe? Current these are all subteams of staged.

Reply:
IIUC we decided not to explicitly list these